### PR TITLE
Feature/cumulative delivery report (WIP)

### PR DESCRIPTION
**Pull request by jofomah**
connects #44

### CODE REVIEW

**Comment by janetbutts** [comment]
@freefony , please review

File: src/app/reports/reports.service.js [hunk 1]
```diff
@@ -1,45 +1,139 @@
...
+				endDate = new Date(endDate).toJSON();
+				var options = {
+					startkey: [startDate],
+					endkey: [endDate, {}, {}]
+				};
+				return dbService.getView(view, options)
+						.then(_this.collateReport);
+			};
+
+			//_this.getByWithin = function(state, startDate, endDate) {
```
Comment (freefony) [inline src/app/reports/reports.service.js]: if this is not useful, maybe we remove it

Comment (jofomah) [inline src/app/reports/reports.service.js]: @freefony this is a work in progress like the title (WIP suggests)

**Comment by jofomah** [comment]
@freefony i think this can be merged in after review, while filtering by lga, ward and state will come in as a separate PR

**Pull request merged**